datalad/interface/tests/test_save.py

```diff
@@ -81,7 +81,9 @@ def test_save(path):
     with open(opj(subds.path, "some_file.tst"), "w") as f:
         f.write("something")
     subds.add('.')
-    ok_clean_git(subds.path, annex=isinstance(ds.repo, AnnexRepo))
+    ok_clean_git(subds.path, annex=isinstance(subds.repo, AnnexRepo))
```

Comment:
That's not what that parameter actually tested. In case `ds` wasn't an annex, it simply wouldn't have checked for `subds` being an annex. At least, that's what `ok_clean_git` did, when the assertion here was written. And that doesn't make sense.